specs/protocol/safe-extensions.md

```diff
@@ -1,41 +1,56 @@
+    - [Liveness Guard Security Properties](#liveness-guard-security-properties)
+    - [Liveness Module Security Properties](#liveness-module-security-properties)
+  - [Interdependency between the Liveness Guard and Liveness Module](#interdependency-between-the-liveness-guard-and-liveness-module)
+- [Operational Considerations](#operational-considerations)
   - [Manual validation of new owner liveness](#manual-validation-of-new-owner-liveness)
-  - [Deploying the liveness checking system](#deploying-the-liveness-checking-system)
-  - [Modify the liveness checking system](#modify-the-liveness-checking-system)
-    - [Replacing the module](#replacing-the-module)
-    - [Replacing the guard](#replacing-the-guard)
+  - [Deploying the Liveness Checking System](#deploying-the-liveness-checking-system)
+  - [Modifying the Liveness Checking System](#modifying-the-liveness-checking-system)
+    - [Replacing the Liveness Module](#replacing-the-liveness-module)
+    - [Replacing the Liveness Guard](#replacing-the-liveness-guard)
 
 <!-- END doctoc generated TOC please keep comment here to allow auto update -->
 
-The Security Council (at
-[eth:0xc2819DC788505Aac350142A7A707BF9D03E3Bd03](https://etherscan.io/address/0xc2819DC788505Aac350142A7A707BF9D03E3Bd03))
-uses a specially extended Safe multisig contract to provide additional security guarantees on top of
-those provided by the Safe contract.
+This document describes extensions to the Security Council and Guardian Safe contracts, which
+provide additional functionality and security guarantees on top of those provided by the Safe
+contract.
 
-## Deputy guardian module
+These extensions are developed using two types of contracts
+([modules](https://docs.safe.global/advanced/smart-account-modules) and
+[guards](https://docs.safe.global/advanced/smart-account-guards)) which the Safe contract has
+built-in support for:
 
-The Security Council acts as the Guardian, which is authorized to activate the [Superchain
-Pause](../protocol/superchain-configuration.md#pausability) functionality and for
-[blacklisting](../fault-proof/stage-one/bond-incentives.md#authenticated-roles) dispute game
-contracts.
+1. **Guard contracts:** can execute pre- and post- transaction checks.
+1. **Module contracts:** a contract which is added to the Safe by the signers, and thenceforth is
+   authorized to execute transactions via the Safe. This means the module must properly implement
+   auth conditions internally.
+
+For more information about the Security Council and Guardian roles, refer to the
+[Stage One Roles and Requirements](./stage-one.md) document.
+
+## Guardian Safe
+
+The Guardian Safe is extended by the Deputy Guardian Module.
+
+### Deputy Guardian Module
+
+As the sole owner of the Guardian Safe, the Security Council acts as the Guardian, which is
+authorized to activate the [Superchain Pause](../protocol/superchain-configuration.md#pausability)
+functionality.
```

Comment:
Although this alludes to stage 1 config, I think it's useful for clarifying the motivation for the DGM.